force-app/main/default/classes/OSF_AccDetailsExtension.cls

```diff
@@ -0,0 +1,53 @@
+public with sharing class OSF_AccDetailsExtension {
+    public OSF_AccDetailsExtension(ApexPages.StandardController stdController) { }
+    
+    public String actid {get; set;}
+    public list <Contact> acc {get;set;}
+    public String searchKey {get;set;}
+    String var = ApexPages.currentPage().getParameters().get('id');
```

Comment:
please use more descriptive name for this and ideally use it inside the related method